doc/en/nose.rst

```diff
@@ -68,4 +68,13 @@ Unsupported idioms / known issues
   fundamentally incompatible with pytest because they don't support fixtures
   properly since collection and test execution are separated.
 
+Migrating from Nose to Pytest
```

Comment:
```suggestion
Migrating from nose to pytest
```

pytest is always spelled  with lower-case P, and I believe it's the same case with nose if the docs are any indication:

https://nose.readthedocs.io/en/latest/testing.html